src/translations/screens/Feedback.ts

```diff
@@ -0,0 +1,16 @@
+import {translation as _} from '../commons';
+
+const GoodOrBadTexts = {
```

Comment:
Greit å bruke samme navn her som blir eksportert globalt

```suggestion
const FeedbackTexts = {
```